gramlib/stream.ml

```diff
@@ -24,7 +24,6 @@ and buffio =
   { ic : in_channel; buff : bytes; mutable len : int; mutable ind : int }
 
 exception Failure
```

Comment:
Why is this kept?

Reply:
This is a flow control exception and not a printable exception